content/kit/_index.md

```diff
@@ -28,19 +28,24 @@ In addition to the boards, your kit will also contain:
 
 | Part                         | Quantity | Specification               | Notes                                  |
 |------------------------------|----------|-----------------------------|----------------------------------------|
+| Battery Bag                  | 1        |                             |                                        |
 | Battery Charger              | 1        | iMAX B6                     |                                        |
 | Battery Charger Power Supply | 1        | 12V 5A                      |                                        |
-| Battery                      | 1        | 11.1V 2.2Ah Lithium Polymer |                                        |
-| USB Memory Stick             | 1        | 1GB                         |                                        |
+| Battery                      | 2        | 11.1V 2.2Ah Lithium Polymer |                                        |
+| Webcam                       | 1        | TeckNet C016                |                                        |
 | USB Hub                      | 1        |                             |                                        |
 | Micro USB Cable              | 2        |                             |                                        |
 | USB B Cable                  | 1        |                             |                                        |
-| Black Wire                   | 3m       |                             | Used for ground *only*                 |
-| Red Wire                     | 3m       |                             |                                        |
-| Blue Wire                    | 3m       |                             |                                        |
-| Small CamCon                 | 2        | 5mm                         |                                        |
 | CamCon                       | 2        | 7.5mm                       |                                        |
+| Small CamCon                 | 2        | 5mm                         |                                        |
 | Micro CamCon                 | 3        | 3mm                         | 1 additionally found on Pi power cable |
```

Comment:
Could we take this opportunity to rename the CamCons a bit more consistently? e.g. "large", "medium", "small" instead of (blank), "small" and "even smaller".